### ISSUE TITLE
rss or feeds (atoms) convert to tiddlers

### ISSUE THREAD

**Issue by Serj-Aleks** [Jermolene/TiddlyWiki5#2550]
I decided to register this, because I believe that this step will increase the number of potential users Tiddlywiki

**Comment by twMat** [comment]
@Serj-Aleks  I agree it is a desirable application but I fear it is much to vague to be actionable and therefore posting it here will just not lead to anything. Considering the overload of issues here, may I propose that you close this issue and instead bring it up on the google boards for discussion?

(Side note: When the so called "TWederation" eventually is functional, I hope to set up a "pre-github bakery" where great ideas like yours can be refined before they're brought up here on github.)

**Comment by Serj-Aleks** [comment]
I understand the general course. If anything, I'm sorry. But I have a better understanding from the perspective of working with the information with its analysis, ie content. So maybe I have a little different priorities. :)
